cmd/vulcan-vulners/main.go

```diff
@@ -105,8 +105,17 @@ func severity(score float32) string {
 	}
 }
 
-// vulnerabilities retrieves the vulnerabilities affecting a software component from the vulners.com API.
-func vulnerabilities(p, s, v, t string) (*report.Vulnerability, error) {
+type vulnerFinding struct {
```

Comment:
Probably OCD but can we replace `vulner` by `vulners`? :)

Reply:
Yes no probs.